docs/admin/install/helm-install/google-gke.md

```diff
@@ -3,6 +3,186 @@ tags:
+    ```
+    kubectl create namespace jans
+    ```
+
+## Jans Installation using Helm
+1.  Install [Nginx-Ingress](https://github.com/kubernetes/ingress-nginx), if you are not using Istio ingress
+    
+      ```
+      helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+      helm repo add stable https://charts.helm.sh/stable
+      helm repo update
+      helm install nginx ingress-nginx/ingress-nginx
+      ```
+
+2.  Create override.yaml file and add changes as per your desired configuration:
+
+    - FQDN/domain not registered:
+    
+        
+        Get the Loadbalancer IP: 
+        ```
+        kubectl get svc nginx-ingress-nginx-controller --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        ```
+
+      
+        
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: #Add IP from previous command
+            isFqdnRegistered: false
+        ```
+
+    - FQDN/domain registered:
+    
+        If the FQDN for jans i.e `demoexample.jans.org` is registered and globally resolvable, you have to map it to the loadbalancer IP created in the previous step by Nginx-Ingress.
+        More details in this [guide](https://medium.com/@kungusamuel90/custom-domain-name-mapping-for-k8s-on-gcp-4dc263b2dabe).
+
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: "" #Add LoadBalancer IP from previous command
+            isFqdnRegistered: true
+            fqdn: demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        nginx:
+          ingress:
+              enabled: true
+              path: /
+              hosts:
+              - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+              tls:
+              - secretName: tls-certificate
+                hosts:
+                - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        ```
+
+
+
+
+
+
+    -  LDAP/Opendj for persitence storage
+
+
+          Add this to override.yaml:
+          ```yaml
+          global:
+            cnPersistenceType: ldap
+            storageClass:
+              provisioner: kubernetes.io/gce-pd
+            opendj:
+              enabled: true
+          ```
+
+          So now configuring both LDAP and no-FQDN will look something like that:
+
+          ```yaml
+           global:
+             cnPersistenceType: ldap
+             lbIp: #Add Load Balancer IP from previous command
+             isFqdnRegistered: false
+             storageClass:
+               provisioner: kubernetes.io/gce-pd
+             opendj:
+               enabled: true
+          ```
+
+
+
+
+
+
+
+    - MySQL for persistence storage
+      ```
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install my-release --set auth.rootPassword=Test1234#,auth.database=jans bitnami/mysql -n jans
+      ```
+
+        Create a file named `mysql.yaml` and this to it:
```

Comment:
if you are naming this `mysql.yaml` hinting to the persistence used you should follow the same convention with LDAP. Call the file `jans-override-values.yaml` in all occasions so that at the end of this doc the helm install points to the right override file